packages/client3/components/design-system/input/checkbox.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+
+import { Checkbox } from "./checkbox";
+
+export default {
+  component: Checkbox,
+  title: "Components/Input/Checkbox",
+} as ComponentMeta<typeof Checkbox>;
+
+export const CheckboxStory: ComponentStory<typeof Checkbox> = (args) => (
+  <Checkbox {...args} />
```

Comment:
When testing this component I get:
<img width="1719" alt="image" src="https://user-images.githubusercontent.com/14227409/189339035-168ac1f9-e068-4289-93fc-352cd14aaf3d.png">
Is this expected?

Reply:
Yeah, It's because for `checkbox control` the state is controlled by the `react hook form` in actual implementation.  When used along with hook form, it will take care of adding those required props